### DISCUSSION TITLE
phone auth error with app that deplay to google play store, but working fine with release .apk from build

### DISCUSSION THREAD

**Discussion by somchithDroppin** [invertase/react-native-firebase#4208] [Q&A] [answered]
This is a message from error : [auth/app-not-authorized] this app not authorized to use Firebase Authentication, Please verify that the correct package name and SHA-1 are configured in the Firebase Console. [App validation failed].

Please help if I miss something,
Thank you very much.
![2ac2a9c0-c600-4b06-81c7-4f91e920edc9](https://user-images.githubusercontent.com/55729507/92201167-07bc8a80-eea6-11ea-8cad-d2ad46680f3c.jpeg)

**Comment by mikehardy** [comment, marked as answer]
The error message is quite descriptive.
You may have your release build set up correctly but your debug build is not, and the error message indicates why. It has a different SHA-1
